src/ml/neural_net/style_transfer/mps_style_transfer.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+#import <ml/neural_net/mps_descriptor_utils.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(macos(10.15))
+@interface TCMPSStyleTransfer : NSObject
+
+@property (nonatomic) NSUInteger batchSize;
+@property (nonatomic) NSNumber *contentLossMultiplier;
```

Comment:
Any reason to prefer `NSNumber` over `float`?

Reply:
Not particularly. Seemed more Objective-C like.

Reply:
Most ObjC developers will avoid `NSNumber` when possible. It only exists when object semantics are required, e.g. when creating an `NSArray` of numbers.

Reply:
Sounds good, switching to float.